x-pack/plugins/ingest_manager/public/applications/ingest_manager/constants/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export {
+  PLUGIN_ID,
+  EPM_API_ROUTES,
+  DEFAULT_AGENT_CONFIG_ID,
+  AGENT_CONFIG_SAVED_OBJECT_TYPE,
+} from '../../../../common';
+
+// intially tried `/app/${PLUGIN_ID}` but that returned `/app/undefined`
```

Comment:
Really small typo `intially` -> `Initially`

I also ran into this issue at some point during migration and I think it is due to some sort of import/export dependency order issue 🤷🏻‍♀️ not a big deal, and gives us room to make the UI base path prettier in the future (like `/app/ingest`)